geopolars/src/geoseries.rs

```diff
@@ -550,6 +686,139 @@ mod tests {
         assert_eq!(result, correct, "Should get the correct convex hull");
     }
 
+    #[test]
+    fn skew() {
+        let geo_series = Series::from_geom_vec(&[Geometry::Polygon(polygon!(
+        (x: 0.0,y:0.0),
+        (x: 0.0,y:1.0),
+        (x: 1.0,y: 1.0),
+        (x: 1.0,y: 0.0)
+        ))])
+        .unwrap();
+
+        let result: Geometry<f64> = Geometry::Polygon(polygon!(
+        (x:-0.008727532464108793,y:-0.017460384745873865),
+        (x:0.008727532464108793,y:0.9825396152541261),
+        (x:1.008727532464109, y:1.0174603847458739),
+        (x:0.9912724675358912, y:0.017460384745873865)
+        ));
+
+        let skewed_series = geo_series.skew(1.0, 2.0, TransformOrigin::Center);
+        assert!(skewed_series.is_ok(), "To get a series back");
+
+        let geom = iter_geom(&skewed_series.unwrap()).next().unwrap();
+        assert_eq!(geom, result, "the swkfddfg");
```

Comment:
More descriptive assertion text?

Reply:
Again think this is an unfortunate past error :-)